src/js/views/hallOfFame.js

```diff
@@ -30,11 +30,19 @@ async function updatePlayers(inputs, updateEvents) {
                 min: 0,
                 per: 0,
             };
+            players[i].teamSums = {};
             for (let j = 0; j < players[i].stats.length; j++) {
+                const team = players[i].stats[j].tid;
                 if (players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per > players[i].bestStats.gp * players[i].bestStats.min * players[i].bestStats.per) {
                     players[i].bestStats = players[i].stats[j];
                 }
+                if (players[i].teamSums.hasOwnProperty(team)) {
+                    players[i].teamSums[team] += players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per;
```

Comment:
> Could you quickly explain the 67 and 30 in the denominator there? What are they referencing?

I just copied them from the footnote of http://insider.espn.go.com/nba/hollinger/statistics and then added the 0.8 factor because the values seemed unreasonably high (although I haven't recently checked if that's still true).

> By PER, the 2019 MIN season is the best, despite having lower EWA than the 2021 NYC season. I'm really curious why this is the case

See the formula in the link above, the constant subtracted from PER makes it so a PER of 11.05 (for FCs) gives an EWA of 0. Which kind of makes sense if you think about it. Like if I have a PER of 10 (bad!) and I play tons of minutes, I do not produce the same amount of wins as someone with a PER of 20 (really good!) who plays half of my minutes.

(Interestingly, PER itself falls prey to this kind of problem.... I forget the exact cutoff point, but even really bad shooters can increase their PER simply by shooting more.)

So overall, using EWA from this formula is probably more informative than GP * MPG * PER. Although both are still pretty shitty advanced stats, I just use them because they are simple and relatively fast to compute.